contracts/tokens/ERC721EnumerableImplementation.sol

```diff
@@ -14,11 +14,6 @@ contract ERC721EnumerableImplementation is ERC721implementation {
    */
   uint256[] internal tokens;
 
-  /*
```

Comment:
We decided to implement a basic _burn function so this mappings will need to stay.

Reply:
We should put it back when we re-implement burn.

Reply:
Ok, #13 has burn implemented.